src/screens/CreatePost/index.js

```diff
@@ -828,10 +825,7 @@ const CreatePost = () => {
             </>
           )}
           <Gap style={styles.height(25)} />
-          <Button
-            disabled={isButtonDisabled()}
-            onPress={postV2}
-            style={styles.buttonPost(useDynamicColors(typeUser))}>
+          <Button disabled={isButtonDisabled()} onPress={postV2}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The style prop has been removed from the Button component. If the button relied on dynamic coloring, make sure it's handled differently now.
```diff
- style={styles.buttonPost(useDynamicColors(typeUser))}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->